AVPlayerExample/ViewController.m

```diff
@@ -195,31 +201,12 @@ - (void)prepareMedia {
     [self startPreview];
 }
 
-- (void)setupAudioSession {
-    // In this example we don't want TwilioVideo to dynamically configure and activate / deactivate the AVAudioSession.
-    // Instead we will setup audio once, and deal with activation and de-activation manually.
-    [[TVIAudioController sharedController] configureAudioSession:TVIAudioOutputVideoChatDefault];
-
-    // This is similar to when CallKit is used, but instead we will activate AVAudioSession ourselves.
-    NSError *error = nil;
-    [[AVAudioSession sharedInstance] setActive:YES error:&error];
-    if (error) {
-        [self logMessage:[NSString stringWithFormat:@"Couldn't activate AVAudioSession. %@", error]];
-    }
-
-    [[TVIAudioController sharedController] startAudio];
```

Comment:
Do we still want to activate the AVAudioSession ourselves? I guess AVPlayer will do it for us, but it might make things more explicit if we do it.

Reply:
As discussed, we will not add AVAudioSession activation/deactivation logic here.